### PR TITLE
[IOTDB-2214] InfluxDB-Connect write point time error format

### PR DESCRIPTION
**Pull request by GOptimistic**
see https://issues.apache.org/jira/browse/IOTDB-2214

### CODE REVIEW

**Comment by SteveYurongSu** [comment]
Hi @GOptimistic, nice to see u here!

Let's add some tests for the PR?

File: server/src/main/java/org/apache/iotdb/db/utils/TimeUtil.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.iotdb.db.utils;
+
+import java.util.concurrent.TimeUnit;
+
+public class TimeUtil {
```
Comment (SteveYurongSu) [inline server/src/main/java/org/apache/iotdb/db/utils/TimeUtil.java:23]: Move it to DateTimeUtils

**Review by SteveYurongSu — COMMENTED** [review]
Overall LGTM

**Comment by jun0315** [comment]
@GOptimistic Great! Thanks a lot for your contribution!

**Review by SteveYurongSu — APPROVED** [review]

**Pull request merged**